src/components/App.tsx

```diff
@@ -66,6 +83,7 @@ export default function App(): JSX.Element {
                 render={() => LEVELS[levelUrl]}
               ></Route>
             ))}
+            <Route exact path="/leaderboard" render={() => <Leaderboard />} />
```

Comment:
whats the difference between this and 

`  <Route exact path="/leaderboard" render={Leaderboard} /> `

is it the React state?

Reply:
React doesn't look at render={Leaderboard} as a functional component so you can't use context hooks in it Leaderboard.

Reply:
okay thats what i thought too